### PR TITLE
fixed naming convention, spaces, etc

### PR DESCRIPTION
**Pull request by eduardogamboa11**

### CODE REVIEW

File: account/views.py [hunk 1]
```diff
@@ -146,47 +160,57 @@ def removeAccount(request):
...
     return JsonResponse(response)
 
+
 @require_POST
 @csrf_exempt
 def getProjects(request):
     if request.method == 'POST':
-        projects = get_object_or_404(Account, id = request.POST['account']).projects.all()
+        projects = get_object_or_404(Account, id=request.POST['account']).projects.all()
         idProjects = []
```
Comment (mmartinezintersys) [inline account/views.py]: Should be, idProjects => id_projects

File: search/views.py [hunk 2]
```diff
@@ -36,49 +39,57 @@ def search(request):
...
-            total += consultant_skills[bio.name][tag]['skill_ocurrence']
-            if consultant_skills[bio.name][tag]['skill_ocurrence']:
-                skills[tag] = consultant_skills[bio.name][tag]['skill_ocurrence']
+            total += consultant_skills[bio.name][tag]['skill_occurrence']
+
+            if consultant_skills[bio.name][tag]['skill_occurrence']:
+                skills[tag] = consultant_skills[bio.name][tag]['skill_occurrence']
+
         if total:
             availability, days_until_available, utilisation, projects = get_availability(bio)
```
Comment (mmartinezintersys) [inline search/views.py]: utilisation => utilization

File: search/views.py [hunk 3]
```diff
@@ -118,7 +131,6 @@ def get_availability(bio):
     if len(projects) is 0:
         days_until_available = -1
         availability = 'Available'
-
     else:
         for project in projects:
             total_utilisation += project.utilisation
```
Comment (mmartinezintersys) [inline search/views.py]: total_utilisation => total_utilization

File: consultantmarket/views.py [hunk 4]
```diff
@@ -3,81 +3,91 @@
...
 from django.conf import settings
-from django.shortcuts import redirect
 from django.contrib import messages
+from django.http import HttpResponseRedirect
+from django.shortcuts import render, redirect
+from django.views.decorators.csrf import csrf_exempt
 from django.views.decorators.http import require_POST
 from django.core.mail import EmailMultiAlternatives
 
 from consultantmarket import redirect_url
```
[the remark this answers is not in the document]
Comment (eduardogamboa11) [reply consultantmarket/views.py]: Utilisation is used with s, as that is how it comes from kimble, utilisation is british version, perhaps changing to z shouldnt be as necessary, confusing perhaps.

**Comment by eduardogamboa11** [comment]
will rework task.

**Pull request closed without merging**